lib/confluence/api_velocity.es6.js

```diff
@@ -38,88 +40,87 @@ foam.CLASS({
       },
     },
   ],
+
   methods: [
+    function run() {
```

Comment:
It was because I thought it was documented in a super class. (`function methodName() { ...}` is the shorthand method syntax when no other metadata required.) However, this isn't a `MetricComputer` *per se*; we probably should have a `Runnable` interface implemented by `MetricComputer` (and its subclasses) and by `ApiVelocity`. For now, I've copied the doc from `MetricComputer` in here.